ChangeLog.d/fix-parllel-cmake-build-fail.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+   * Fix a build bug : when parallel out-of-box build with cmake, it raise file
+     not found error. That's due to same target is included in different sub-
+     makefiles. And cmake will call the targets in different forked `make`
+     process. As a result, the dependancy will be broken. This fix remove the
+     targets in sub-makefiles and create a global target for it.
```

Comment:
1. This is about out-of-*source* builds (a.k.a. out-of-tree builds), not out-of-box builds.
1. This is too much information. The changelog is for users. It should explain the symptoms of the problem, not the internal details.
1. Changelog entries that fix a bug should mention the bug number.
1. Changelog entries are very public and should be in good English.

Suggested text:

> Fix a race condition in out-of-source builds with CMake when generated data files are already present. Fixes #5374,